draft-rfcs/working-group-terminology.md

```diff
@@ -16,6 +16,8 @@ Working groups were not created through the RFC process, as such there's not muc
 
 While this definition is true, it does not define the difference between what has becomed to be called "Domain Working Groups" and "Lang team Working Groups", additionally the community has also adopted the Working Group terminology for working on areas they are interested in, independent of The Rust Programming Language's Working Groups. It's great that we have a community able to self organise in this way, however it has led to some confusion over who is supporting these efforts, and whether they're _"official"_ Working Groups.
```

Comment:
As well as 'lang team working groups' there are WGs for other teams, e.g., dev-tools had a few and I think still does. I think other teams did too.

Reply:
Yes, initially used lang team working groups as the example lang team is most prominent example. I'll make it more general.